### ISSUE TITLE
Properties file breaks while parsing line ending with "\" (line breaker)

### ISSUE THREAD

**Issue by ajays20078** [ssato/python-anyconfig#56]
Contents of `one.properties`

```cat one.properties 
key=\
val1,\
val2,\
val3
```

```>>> import anyconfig
>>> a = anyconfig.load("one.properties")
>>> a
{'key=key=val1,key=key': 'val1,val2,val3'}
```

Where as it works well with other properties parser.

```>>> from pyjavaproperties import Properties
>>> p = Properties()
>>> p.load(open('one.properties'))
>>> p
<pyjavaproperties.Properties object at 0x107fac1d0>
>>> p.list()
-- listing properties --
key=val1,val2,val3
>>> print p
{key=val1,val2,val3}
```

Line break is causing issue with anyconfig parser.

**Comment by ajays20078** [comment]
Moving [this](https://github.com/ssato/python-anyconfig/blob/0a39a47e43dc999f4979260ccabac5c9c19408da/anyconfig/backend/properties.py#L177) line before [this](https://github.com/ssato/python-anyconfig/blob/0a39a47e43dc999f4979260ccabac5c9c19408da/anyconfig/backend/properties.py#L173) line  might fix it?

**Comment by ssato** [comment]
Excuse me to late response. 

Thanks a lot for you reports and fix suggestion!
Could you please wait for a while?I want to look closely into the causes.

**Comment by ssato** [comment]
I confirmed that you're right and commit the fix b804c61. Thanks a lot!

**Comment by ajays20078** [comment]
Thanks! yeah its works. Closing the issue.
